css/sitesummary.css

```diff
@@ -26,19 +23,34 @@
     margin-bottom: 12px;
 }
 
-.package-summary table.ss-gridfield-table tr td.col-Summary {
-    padding: 0;
-}
-
 a.package-summary__anchor {
```

Comment:
Considering `package-summary` has been changed to `site-summary` you should remain consistent.

Reply:
No, this is a separate selector. It summarises a package (as the block*), not the site (as the gridfield does) - the site level summary being the one I updated.

*Block as the B in BEM.